tests/tasks/test/integration/migrate.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#MISE description="Test Prometheus metrics are exported and updated"
+
+#!/bin/bash
+
+set -e
+
+
+docker exec -i postgres${CONTAINER_SUFFIX} psql postgresql://cipherstash:password@proxy:6432/cipherstash <<-EOF
+TRUNCATE encrypted;
+EOF
+
+# Connect to the proxy
+docker exec -i postgres${CONTAINER_SUFFIX} psql postgresql://cipherstash:password@proxy:6432/cipherstash <<-EOF
+INSERT INTO encrypted (id, plaintext) VALUES (1, 'One');
+INSERT INTO encrypted (id, plaintext) VALUES (2, 'Two');
+INSERT INTO encrypted (id, plaintext) VALUES (3, 'Three');
+INSERT INTO encrypted (id, plaintext) VALUES (4, 'Four');
+INSERT INTO encrypted (id, plaintext) VALUES (5, 'Five');
+EOF
+
+docker exec -i proxy cipherstash-proxy encrypt --table encrypted --columns plaintext=encrypted_text  --verbose
+
+docker exec -i postgres${CONTAINER_SUFFIX} psql postgresql://cipherstash:password@proxy:6432/cipherstash <<-EOF
+SELECT * FROM encrypted;
+EOF
+
+set +e
+OUTPUT="$(docker exec -i postgres${CONTAINER_SUFFIX} psql 'postgresql://cipherstash:password@proxy:6432/cipherstash?sslmode=disable' --command 'SELECT id FROM encrypted WHERE encrypted_text IS NULL' 2>&1)"
```

Comment:
Besides this non-null test, it would be great to also check the encrypted columns contain the expected values, perhaps with something like checking `WHERE encrypted_text = 'Three'` returns a row